From 3f00e594d7536274a612ecf157349c60aeb53064 Mon Sep 17 00:00:00 2001
From: ArenM <aren@peacevolution.org>
Date: Wed, 27 Apr 2022 23:57:38 -0400
Subject: [PATCH sxmo-utils 1/2] avoid conflicting with systemd services

On arch we manage run mmsd and vvmd as systemd user services, so we need
to modify the sxmo scripts to use systemd instead of superd for them.
---
 scripts/core/sxmo_mmsdconfig.sh | 3 +++
 scripts/core/sxmo_vvmdconfig.sh | 3 +++
 2 files changed, 6 insertions(+)

diff --git a/scripts/core/sxmo_mmsdconfig.sh b/scripts/core/sxmo_mmsdconfig.sh
index b923c78..0650439 100755
--- a/scripts/core/sxmo_mmsdconfig.sh
+++ b/scripts/core/sxmo_mmsdconfig.sh
@@ -7,6 +7,9 @@
 # shellcheck source=scripts/core/sxmo_common.sh
 . sxmo_common.sh
 
+# on arch we manage mmsd with systemd instead of superd
+alias superctl='systemctl --user'
+
 set -e
 
 MMS_BASE_DIR="${SXMO_MMS_BASE_DIR:-"$HOME"/.mms/modemmanager}"
diff --git a/scripts/core/sxmo_vvmdconfig.sh b/scripts/core/sxmo_vvmdconfig.sh
index 6cfab48..12fb581 100755
--- a/scripts/core/sxmo_vvmdconfig.sh
+++ b/scripts/core/sxmo_vvmdconfig.sh
@@ -9,6 +9,9 @@
 
 set -e
 
+# on arch we manage mmsd with systemd instead of superd
+alias superctl='systemctl --user'
+
 VVM_BASE_DIR="${SXMO_VVM_BASE_DIR:-"$HOME"/.vvm/modemmanager}"
 VVMCONFIG="$VVM_BASE_DIR/vvm"
 
-- 
2.43.0

